test/fixtures/all.env

```diff
@@ -1,6 +1,9 @@
 #
 # env file for testing that contains all possible and impossible values
 #
+# ignored
+AEMCOMPANIONPATH=foobar
+AEM_COMPANIONPATH=foobar
```

Comment:
silly question: why is `AEM_COMPANIONPATH` ignored?

Reply:
we don't use the logic of yargs to require arguments, but we check it _manually_:
https://github.com/adobe/helix-cli/blob/e274252a6ec8ffadd8460257d383a8504bcbe793/src/cli.js#L36-L44

this was exactly due to this problem of different programs using the same prefix. but I think yargs changed something in this respect, so that `env('AEM')` would match `AEM*` and `AEM_*`. before it was only with the `_`